### ISSUE TITLE
Docbook output can produce invalid manpages

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#143]
Originally reported of Google Code with ID 87.

```
Sometimes the docbook man generator doesn't quote things properly. In the
2.0.6 release, man/gnt-cluster.8.in contains at line 363 three dots (...)
which were folded such that they appear at the beginning of the line. This
is wrong, since dot at beginning of line is a macro call in roff language,
so this "eats" the rest of the line (since the macro ".." is unknown).

The distcheck rule should check that:
  LANG=C MANWIDTH=80 man --warnings -E UTF-8 -l manpage-file >/dev/null

on each .[78].in file doesn't produce any warnings.

For now the debian package will just patch the file; other people will see
a wrong snippet in the man page.
```
Originally added on 2010-02-09 19:10:25 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in commit a4f12da4541f308198061f3e046c3f3e872f79c4.
```
Originally added on 2010-02-10 10:54:10 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
